src/components/refresh-button/refresh-button.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import { theme } from 'style/theme';
+import { FONT } from 'style/typography';
+import { orange, darkOrange } from 'style/colors';
+
+export const RefreshButton = styled.button`
```

Comment:
Most of these are not required if you've used Mui-Button, try to minimize the CSS required.